config/cloudfront/live.json

```diff
@@ -5651,10 +5558,68 @@
                         "contrastMode",
                         "blf-alpha-session",
                         "_csrf",
-                        "blf-ab-1-50",
                         "blf-ab-afa"
                     ],
-                    "Quantity": 5
+                    "Quantity": 4
+                }
+            },
+            "QueryStringCacheKeys": {
+                "Items": [],
+                "Quantity": 0
+            },
+            "QueryString": false
+        },
+        "MaxTTL": 31536000,
+        "PathPattern": "/welsh/home",
```

Comment:
How come this is getting added?

Reply:
Any alias listed for a route automatically gets a Welsh version added too. Kind of redundant in this case but causes no harm really.

Reply:
👍